packages/@aws-cdk/aws-route53-targets/README.md

```diff
@@ -48,5 +41,26 @@ This library contains Route53 Alias Record targets for:
     // or - route53.RecordTarget.fromAlias(new alias.ApiGatewayDomainName(domainName)),
   });
   ```
+* S3 Bucket WebSite:
+
+**Important:** The Bucket name must strictly match the full DNS name. 
```

Comment:
I'd like to further improve the user experience and check in `aws-route53` that the Bucket name matches `RecordSet` and `HostedZone`. How do you guys feel about that?

Reply:
I like it!

Reply:
I'll make another PR for it 👍